src/opnsense/mvc/app/models/OPNsense/Interfaces/VxLan.php

```diff
@@ -28,8 +28,56 @@
 
 namespace OPNsense\Interfaces;
 
+use Phalcon\Messages\Message;
 use OPNsense\Base\BaseModel;
 
 class VxLan extends BaseModel
 {
+    public function performValidation($validateFullModel = false)
+    {
+        $messages = parent::performValidation($validateFullModel);
+
+        foreach ($this->vxlan->iterateItems() as $vxlan) {
+
+            $vxlangroup = (string) $vxlan->vxlangroup;
+            $vxlanremote = (string) $vxlan->vxlanremote;
+            $vxlandev = (string) $vxlan->vxlandev;
+
+            // Validation 1: At least one of vxlangroup and vxlanremote must be populated, but not both.
+            if ((!empty($vxlangroup) && !empty($vxlanremote)) ||
+                (empty($vxlangroup) && empty($vxlanremote))) {
+                $messages->appendMessage(new Message(
+                    "Remote address -or- Multicast group has to be specified",
```

Comment:
If you wrap this in (e.g. `gettext('"Remote address -or- Multicast group has to be specified"')`), it should be collected using our crawler for translations. You wouldn't notice a difference yet, but after the phrase is being translated via our translation project (https://translate.opnsense.org/), it would return the correct language.

Reply:
I added the gettext function to all messages.